bin/format-cpan-uploads

```diff
@@ -142,14 +121,13 @@ open my $md_fh, '>:utf8', sprintf 'what-s-new-on-cpan---%s-%s.md',
 my $metadata = {
   title       => sprintf('What\'s new on CPAN - %s %s', $datetime->fullmonth, $datetime->year),
   description => sprintf('A curated look at %s\'s new CPAN uploads', $datetime->fullmonth),
-  authors     => ['david-farrell'],
-  tags        => [],
+  authors     => ['mathew-korica'],
+  tags        => ['new'],
   categories  => 'cpan',
-  image       => "",
-  draft       => JSON::true,
+  draft       => JSON::false,
   date        => $now->datetime,
-  image       => undef,
-  thumbnail   => undef,
+  image       => "/images/176/2A6DE1D0-0ACE-11E5-A57F-EAC87F6D3C83.png",
+  thumbnail   => "/images/176/thumb_2A6DE1D0-0ACE-11E5-A57F-EAC87F6D3C83.png",
```

Comment:
👍🏻 It looks like David was switching between colours in his posts, but I think this is good.